CommonCPPCTest.cmake

```diff
@@ -82,11 +82,17 @@ macro(common_add_cpp_test NAME FILE)
   add_test(NAME ${TEST_NAME}
     COMMAND ${RUN_PREFIX} $<TARGET_FILE:${TEST_NAME}> ${RUN_ARGS})
 
+  add_custom_target(ctest_${TEST_NAME}
```

Comment:
are set_target_properties(  EXCLUDE_FROM_DEFAULT_BUILD ON FOLDER "Tests" ) not needed for these targets ?